kubernetes/kube.libsonnet

```diff
@@ -163,7 +163,13 @@
     items: std.filter($.isNotNull, $.objectValues(self.items_)),
   },
 
-  Namespace(name): $._Object('v1', 'Namespace', name),
+  Namespace(name): $._Object('v1', 'Namespace', name) {
+    metadata+: {
+      annotations+: {
+        'argocd.argoproj.io/sync-wave': '-10',
```

Comment:
Do we have a doc with the sync waves we have today and their expected usage of them? If so, we should make sure that we update that. If we don't, maybe create a quick one under the ArgoCD docs?

Reply:
Good idea

Reply:
https://outreach-io.atlassian.net/wiki/spaces/DT/pages/2360541948/ArgoCD+sync-wave+table